mcm/json_layer/request.py

```diff
@@ -3009,6 +3009,17 @@ def request_to_tasks(self, base, depend):
         settings_db = database('settings')
         __DT_prio = settings_db.get('datatier_input')["value"]
         validation_info = []
+
+        # Filter efficiency was not check on validation
+        filter_efficiency = self.get_efficiency()
+        request_skip_check = self.get_allowed_skip_validation_check().get(self.get_attribute("prepid"))
```

Comment:
from the code in the PR and the failing check on https://cms-pdmv-prod.web.cern.ch/mcm/public/restapi/requests/get_dict/SUS-RunIISpring21UL16FSGSPremixLLPBugFix-00071 I can only think of this line failing silently